app/javascript/components/explore/DifferentialExpressionPanel.jsx

```diff
@@ -31,26 +36,146 @@ function initChecked(deGenes, checkedGene) {
   return checked
 }
 
+function DownloadButton({bucketId, deFilePath}) {
+  return (
+    <a className="de-download-button"
+      onClick={async () => {await downloadBucketFile(bucketId, deFilePath)}}
+      data-analytics-name="differential-expression-download"
+      data-toggle="tooltip"
+      data-original-title="Download all DE genes data for this group"
+    >
+      <FontAwesomeIcon icon={faDownload}/>
+    </a>
+  )
+}
+
+function DifferentialExpressionTable({
+  genesToShow, searchGenes, checked, clusterName, annotation, species, changeRadio
+}) {
+  return (
+    <>
+    <table className="de-table table table-terra table-scp-compact">
+      <thead>
+        <tr>
+          <th>Name</th>
+          <th>
+            <span className="glossary" data-toggle="tooltip" data-original-title="Log (base 2) of fold change">
+              log<sub>2</sub>(FC)
+            </span>
+          </th>
+          <th>
+            <span className="glossary" data-toggle="tooltip" data-original-title="p-value adjusted with Benjamini-Hochberg FDR correction">
+              Adj. p-value
+            </span>
+          </th>
+        </tr>
+      </thead>
+      <tbody>
+        {genesToShow.map((deGene, i) => {
+          return (
+            <tr className="de-gene-row" key={i}>
+              <td>
+                <label
+                  title="Click to view gene expression.  Arrow down (↓) and up (↑) to quickly scan."
+                ><input
+                    type="radio"
+                    checked={checked[deGene.name]}
+                    data-analytics-name="selected-gene-differential-expression"
+                    value={deGene.name}
+                    onClick={event => {
+                      searchGenes([deGene.name])
+
+                      // Log this search to Mixpanel
+                      const rank = i
+                      logSearchFromDifferentialExpression(
+                        event, deGene, species, rank,
+                        clusterName, annotation.name
+                      )
+
+                      changeRadio(event)
+                    }}/>
+                  {deGene.name}</label></td>
+              <td>{deGene.log2FoldChange}</td>
+              <td>{deGene.pvalAdj}</td>
+            </tr>)
+        })}
+      </tbody>
+    </table>
+    <a href="https://forms.gle/qPGH5J9oFkurpbD76" target="_blank" title="Take a 1 minute survey">
```

Comment:
I think this would benefit from an icon indicating that clicking it will open a new tab, the one that is the square with an arrow

Reply:
This might be worth a design discussion.  Looking into this, I found:

* UK.GOV recommends not using the icon, for [reasons they explain](https://designnotes.blog.gov.uk/2016/11/28/removing-the-external-link-icon-from-gov-uk/).
* Other single-cell products don't use the icon for such links, see e.g. [CELLxGene](https://cellxgene.cziscience.com/) and [10x Genomics](https://www.10xgenomics.com/).
* Major technology companies like [Google](https://blog.google/outreach-initiatives/diversity/lunar-new-year-2023/?_ga=2.25771709.1095051699.1675451359-452518495.1675451359), [YouTube](https://blog.youtube/creator-and-artist-stories/5-creator-ceos-entrepreneur-brand-building-on-youtube/), [Apple](https://www.apple.com/newsroom/2023/01/apple-music-launches-rihannas-road-to-halftime-ahead-of-super-bowl-lvii/) omit the icon in such links.
  * [Microsoft](https://blogs.microsoft.com/blog/2023/01/04/the-future-of-mobility-is-now-five-themes-to-watch-at-ces-2023/), [Amazon](https://aws.amazon.com/blogs/opensource/aws-teams-with-ostif-on-open-source-security-audits/), and [Facebook](https://about.fb.com/news/2023/01/instagram-quiet-mode-manage-your-time-and-focus/) don't open external links in new tabs, and their external links omit the icon
* Terra often omits it, as almost always does SCP.
* The link on production appears the same as this link; this PR doesn't change the link.

If we do add icons for links that open in new tabs, I suggest a ticket to implement it SCP-wide.  I'm not sure it'd be valuable though.  In my opinion, the icon is confusing.  I interpret it as "external link", not necessarily "new tab".  Research linked above finds users get similarly confused by the icon.  And although some web UIs use the icon to indicate links open in a new tab, it seems they don't do so consistently, and most web UIs omit the icon.

Reply:
> The link on production appears the same as this link; this PR doesn't change the link.
Ah that's my bad, I thought this was updated in this PR, indeed a more holistic ticket/review of the use/omission of this icon for SCP seems more appropriate.